AmplifyPlugins/API/AWSAPICategoryPluginIntegrationTests/GraphQL/GraphQLWithUserPoolIntegrationTests/GraphQLWithUserPoolIntegrationTests.swift

```diff
@@ -17,9 +17,23 @@ class GraphQLWithUserPoolIntegrationTests: XCTestCase {
 
     static let amplifyConfiguration = "GraphQLWithUserPoolIntegrationTests-amplifyconfiguration"
     static let awsconfiguration = "GraphQLWithUserPoolIntegrationTests-awsconfiguration"
+    static let credentials = "GraphQLWithUserPoolIntegrationTests-credentials"
+    static var user1: String!
+    static var password: String!
 
     override func setUp() {
         do {
+
+            let credentials = try TestConfigHelper.retrieveCredentials(
```

Comment:
Put this into the static `setUp` since you only have to read the credentials file once